hbase-server/src/main/java/org/apache/hadoop/hbase/client/locking/EntityLock.java

```diff
@@ -218,8 +218,7 @@ public LockHeartbeatWorker(final String desc) {
     }
 
     /**
-     * Shutdown the thread cleanly, quietly. We done.
-     * @return
+     * @return Shutdown the thread cleanly, quietly. We done.
```

Comment:
What does `We done` mean in this context. Minor typo `Shuts down the thread clean and quietly`.